assets/stylesheets/bootstrap/_forms.scss

```diff
@@ -143,7 +143,6 @@ output {
   // HTML5 says that controls under a fieldset > legend:first-child won't be
   // disabled if the fieldset is disabled. Due to implementation difficulty, we
   // don't honor that edge case; we style them as disabled anyway.
-  &[disabled],
   &[readonly],
```

Comment:
How does `readonly` differ from `disabled`? Should it also have the same changes?